src/styles/main.scss

```diff
@@ -0,0 +1,78 @@
+@mixin hand-layout {
+  position: absolute;
+  bottom: 50%;
+
+  border-radius: 5px;
+
+  transform-origin: bottom center;
+}
+
+body {
+  margin: 0;
+  padding-top: 50px;
```

Comment:
I think that moving content using padding is not a good practice. You already have the `page__stop-watch` element. So I suggest adding these 50px to its margin like  `margin: 50px auto;`

Reply:
Done. I decided to center the watch vertically and horizontally.